types/webpack/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for webpack 4.0
+// Type definitions for webpack 4.1
 // Project: https://github.com/webpack/webpack
 // Definitions by: Qubo <https://github.com/tkqubo>
```

Comment:
Mentioning definition owners for review: @tkqubo @bumbleblym @bcherny @tommytroylin @mohsen1 @jcreamer898 @ahmed-taj @alan-agius4 @elliottsj @jason0x43